test/integration/short/timeout-tests.js

```diff
@@ -196,9 +196,9 @@ describe('client read timeouts', function () {
     it('should be used instead of socketOptions.readTimeout and profile.readTimeout for prepared queries executions',
       getMoveNextHostTest(true, true, 3123, 1 << 24, { executionProfile: 'aProfile', readTimeout: 3123 }, profiles()));
     it('should suppress socketOptions.readTimeout and profile.readTimeout when set to 0 for simple queries',
-      getTimeoutErrorExpectedTest(false, false, 3000, { executionProfile: 'aProfile', readTimeout: 0}, profiles()));
-    it('should suppress socketOptions.readTimeout and profile.readTimeout  when set to 0 for prepared queries executions',
-      getTimeoutErrorExpectedTest(true, true, 3000, { executionProfile: 'aProfile', readTimeout: 0}, profiles()));
+      getTimeoutErrorNotExpectedTest(false, false, 1000, { executionProfile: 'aProfile', readTimeout: 0}, profiles()));
```

Comment:
These tests weren't effective previously as we only paused the node for 2 seconds, so we gave it enough time to respond within 3 seconds.